packages/datatrak-web/src/views/Tasks/TaskDetailsPage.tsx

```diff
@@ -36,20 +59,41 @@ export const TaskDetailsPage = () => {
     : '';
 
   const from = useFromLocation();
+
+  const ButtonComponent = () => {
```

Comment:
I think it could be nice to make the ButtonComponent a stand alone component outside of the TaskDetails body and just pass in task as a prop.